src/main/resources/openapi/schemas/reportUseOverTime.json

```diff
@@ -0,0 +1,51 @@
+{
+  "description": "report data entry (agreement line information)",
+  "type": "object",
+  "properties": {
+    "agreementId": {
+      "type": "string",
+      "description": "agreement identifier (UUID)"
+    },
+    "accessCountTimeSlots": {
+      "description": "List of time slots, for example [\"2020-03\", \"2020-04\", \"2020-05\"]",
+      "type": "array",
+      "items": {
+        "type": "string"
+      }
+    },
+    "journals": {
+      "type": "object",
+      "description": "Journals report",
+      "properties": {
+        "totalItemRequests": {
+          "type": "array",
+          "description": "access count per time slot, sum of all journals",
+          "items": {
+            "type": "integer"
+          }
+        },
+        "uniqueItemRequests": {
+          "type": "array",
+          "description": "access count per time slot, sum of all journals",
```

Comment:
Rephrase to "Access counts per period, total for all items""

Reply:
Rephrased to "Access count per period, total for all items"